app/src/main/java/org/mozilla/reference/browser/AppRequestInterceptor.kt

```diff
@@ -26,7 +31,7 @@ class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
                 return RequestInterceptor.InterceptionResponse.Content(page, encoding = "base64")
             }
 
-            else -> context.components.services.accounts.interceptor.onLoadRequest(session, uri)
+            else -> context.components.features.firefoxAccountsFeature.interceptor.onLoadRequest(session, uri)
```

Comment:
I think `services.firefoxAccount` would be better naming here, see comment below about services/backgroundservices.

Reply:
Let's decide later once we have the changes in a-c. `FirefoxAccountsFeature` makes sense, just need to find a home for the interceptor.